spec/videos/video_spec.rb

```diff
@@ -44,7 +44,7 @@
       end
     end
 
-    context 'given a Faraday::ConnectionFailed error' do
+    context 'given an error that causes a Funky::ConnectionError' do
```

Comment:
@bruce-fs You shouldn't test the expected result.

The context should say `'given a SocketError` (which is what you are raising) or more generically a connection error.